etc/httpd/global.conf

```diff
@@ -639,10 +639,10 @@ RewriteRule ^/projects/security/membership-policy.html$ /about/governance/polici
 RewriteRule ^/projects/security/secgrouplist.html$ /about/governance/policies/security-group/ [L,R=301]
 RewriteRule ^/projects/security/security-bugs-policy.html$ /about/governance/policies/security-group/bugs/ [L,R=301]
 
-# bug 818316
+# bug 818316, 1128579
 RewriteRule ^/projects/security/certs(?:/(?:index.html)?)?$ /about/governance/policies/security-group/certs/ [L,R=301]
-RewriteRule ^/projects/security/certs/included(?:/(?:index.html)?)?$ /about/governance/policies/security-group/certs/included/ [L,R=301]
-RewriteRule ^/projects/security/certs/pending(?:/(?:index.html)?)?$ /about/governance/policies/security-group/certs/pending/ [L,R=301]
+RewriteRule ^/(?:projects/security|(\w{2,3}(?:-\w{2})?/)?about/governance/policies/security-group)/certs/included(?:/(?:index.html)?)?$ https://wiki.mozilla.org/CA:IncludedCAs [L,R=301]
```

Comment:
I'm sure this works, but just for readability and simplicity's sake I think it would be better to have these be 2 new rules instead of an update to the existing rules.